modules/ROOT/pages/install-sys-reqs.adoc

```diff
@@ -26,12 +26,10 @@ a different operating system version or distribution, the Runtime Fabric install
 == Hardware Requirements
 Only x86 and x64 architectures are supported.
 
-Runtime Fabric can reserve up to 0.5 cores per worker node for internal services. This is important 
+Runtime Fabric reserves 0.3 cores per worker node for internal services. This is important 
```

Comment:
Should revert this change, and I think we're good.

Reply:
Got it - thank you!